setup.py

```diff
@@ -7,13 +7,6 @@
 
 from setuptools import setup, Command, Extension
 
-# Make sure I have the right Python version.
```

Comment:
Whilst you're here, I think you can safely assume that `print_function` isn't going to be a problem (if someone does try to use py2.7 they'll get a bit more gibberish, but at this point it'll be gibberish either way I think).